### ISSUE TITLE
[Automation - BUG] Business Owner emails are not loading Completely 

### ISSUE THREAD

**Issue by KoteshMCGit** [microsoft/powercat-automation-kit#154]
### Describe the issue

When adding a new project, I was unable to find a large portion of the Business Owners.
![image](https://user-images.githubusercontent.com/107620140/217830573-b0ab0e8b-01fc-4382-9c97-57fad96279aa.png)

### Expected Behavior

Application should show list all the filtered (based on user search keywords) emails to select respective business owner for that automation project.

### What component are you experiencing the issue with?

Automation Project

### What solution version are you using?

1.0.20220913.3

### What app or flow are you having the issue with?

Automation Project

### Steps To Reproduce

Launch the Automation Project application.
1.  Click on "Add Project" button to create new project
2. Provide the respective keyword in business owner column and observe search result is not loading all matched emails.

**Comment by KoteshMCGit** [comment]
As per Application design, we are creating a collection of all emails from office365 user connection. And later we are filtering based on user search keywords (at least 3 chars). 

we have observed a limitation on usage of collections in PowerApps. 

To overcome that limitation, we have changed the design of creating collections. Now we are not creating any collection. When user provides search keywords (at least 3 chars) on UI, immediately fetching the emails from office 365 user connection as search result and populating on UI.
